core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -108,6 +112,7 @@ public class RESTSessionCatalog extends BaseSessionCatalog
   private FileIO io = null;
   private MetricsReporter reporter = null;
   private boolean reportingViaRestEnabled;
+  private CloseableGroup closeableGroup = null;
```

Comment:
Minor: consider using shorter names where possible. The word "group" isn't really helping here, just making the name longer. This could be `closeables`.

Reply:
I was mainly aligning this with the rest of the codebase where we name it `closeableGroup` but `closeables` should be fine as well